src/components/ScheduleGenerate/ScheduleGenerateModal.vue

```diff
@@ -575,6 +577,7 @@ input {
   display: flex;
   width: 246px;
   height: 35px;
+  font-family: 'ProximaNova';
```

Comment:
some of the font-family is 'ProximaNova' and others are 'Proxima Nova' which is what's causing the varying fonts in the modal - i'm pretty sure the one that works for everyone is 'Proxima Nova' * actually i don't know if the font-family is needed here? i think we might have just added it to debug and figure out what's wrong with the font haha so we can prob get rid of it